test/source/tests/browser-recipe.ts

```diff
@@ -70,7 +70,8 @@ export class BrowserRecipe {
   }
 
   public static async pgpBlockVerifyDecryptedContent(t: AvaContext, browser: BrowserHandle, m: TestMessage) {
-    const pgpBlockPage = await browser.newPage(t, `chrome/elements/pgp_block.htm${m.params}`);
+    const pgpHostPage = await browser.newPage(t, `chrome/dev/ci_pgp_host_page.htm${m.params}`);
+    const pgpBlockPage = await pgpHostPage.getFrame(['pgp_block.htm']);
```

Comment:
Clever. You'll need to also likely remove `parentTabId=xxx` from the tests json file, and any other places that call this method.

Reply:
thank you for the detailed feedback!

I'm now trying to understand why `ci_pgp_host_page.htm?account_email=flowcrypt.compatibility%40gmail.com&frame_id=frame_yVMKFLRDiY&message=&has_password=___cu_false___&message_id=162275c819bcbf9b&sender_email=human%40flowcrypt.com&is_outgoing=___cu_false___` works, but `ci_pgp_host_page.htm?account_email=flowcrypt.compatibility%40gmail.com&frame_id=frame_yVMKFLRDiY&message=&hasPassword=___cu_false___&msgId=16a9c109bc51687d&senderEmail=flowcrypt.compatibility%40protonmail.com&isOutgoing=___cu_false___&signature=___cu_true___` throws `TypeError: Cannot read property 'key' of undefined (common/core/pgp-key.js:34:43)`

It seems that in the second case openpgp.js isn't loaded.

Reply:
@tomholub ping

Reply:
Sorry, I don't understand how I'm missing these messages.

Most likely the other test is also trying to use openpgp.js directly somehow. I'll take a look.